### ISSUE TITLE
Problem when trying to run docker container run --publish 80:80

### ISSUE THREAD

**Issue by robkremers** [docker/for-mac#3476]
- [ ] I have tried with the latest version of my channel (Stable or Edge)
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Information

  - macOS Version:

### Diagnostic logs

```
Docker for Mac: version...

```

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by robkremers** [comment]
I am working in MacOS on a recent Apple laptop (2018).
.bash_profile:
export PATH=/opt/local/bin:/opt/local/sbin:$PATH
export PATH=/Applications/Docker.app/Contents/MacOS/:/Applications/Docker.app/Contents/MacOS/Docker:$PATH

if [ -d /Applications/Docker.app/Contents/Resources/etc/ ]; then
  . /Applications/Docker.app/Contents/Resources/etc/docker.bash-completion
  . /Applications/Docker.app/Contents/Resources/etc/docker-machine.bash-completion
  . /Applications/Docker.app/Contents/Resources/etc/docker-compose.bash-completion
fi

When trying to run: 
$ docker container run --publish 80:80 nginx
The website is up, but when refreshing I do not see any refresh log.

When trying to run:
$ docker container run --publish 80:80 --detach nginx
The nginx process is supposed to run in the background. In otherwords the command line should be available again. However this is not the case.
Also when trying to run a docker command, e.g. $ docker info, I get a pop up: Fatal error Docker already running. 
Basically nothing works.
I'm a bit stuck now.

**Comment by robkremers** [comment]
In order to be able to use:
$docker info
I need to take kill all docker processes, close the unix terminal, start a new unix terminal take the docker directory out of $PATH and then I can use the commands again.

**Comment by robkremers** [comment]
Solution to my problem: I should not mention the docker directory in parameter PATH.
That's contrary to what I'm used to.
Sorry for bothering.
